### ISSUE TITLE
Developers telegram or other chat?

### ISSUE THREAD

**Issue by sp00x** [iitc-project/ingress-intel-total-conversion#1304]
Is there a plugin developers TG or chat elsewhere now after the one affiliated with the original IITC project shut down after G+ disappeared? I know IITC-CE has a news feed but that's not it..

**Comment by modos189** [comment]
I want to run https://www.reddit.com/r/IITC/
In one of the next news, I'd like to write about him.
But I don't have much experience with Reddit. It would be great if I could find an experienced moderator.

**Comment by MysticJay** [comment]
I'd prefer a TG-Chat, but I know that RU has some problems with it,

**Comment by johnd0e** [comment]
TG chat is great, but there is no ability to divide discussion between topics, and if some message is not answered immediately - it just get drown under new posts.

**Comment by andrewfhou** [comment]
What about Gitter?

**Comment by johnd0e** [comment]
> What about Gitter?

Or slack, or discord, etc.
There are a lot of alternatives, but why not start from reddit?
https://www.reddit.com/r/IITC/

**Comment by andrewfhou** [comment]
> Or slack, or discord, etc.
> There are a lot of alternatives, but why not start from reddit?
> https://www.reddit.com/r/IITC/

Fair enough. I'm subbed, just spitballing messaging systems that would work/are popular

**Comment by sp00x** [comment]
I guess in relation to topic separation (i.e. threads) then reddit might be a good candidate.. only problem there is that sometimes clients with be more focused on "hot/popular" posts, and hide/down-prioritize the new ones, so one has to remember to set the correct mode in their client for the type of subreddit it is.. 

(subbed for now)

**Comment by johnd0e** [comment]
@modos189
How about adding comments to the  channel's posts? (@discussbot)
Or attach 'Discuss' chat?
(https://telegram.org/blog/privacy-discussions-web-bots)

**Comment by MysticJay** [comment]
I am drowning in ChatAPPs and I hate it!
Several Ingress communities have tested SLACK. But the limit of 10.000 posts really is the culprit.
Most Ingress communities/agents have switches to Telegram and offering a support and a dev channel on TG will not only improve the usage of IITC-CE, it will create more activity in development as well.
When ever new agents complain about the stock intel they still get sent the link to the abandoned IITC. Even Operators still do not know about IITC-CE.
Thus it is a kind of marketing for IITC-CE.

@johnd0e I understand your wish to have a way to stick to a topic in a chat, but we are not developing for us, but for the users. And If IITC-CE keeps to be a niche solution, not broadly accepted as the future IITC successor I will likely drop out of the project sooner than later.

My personal skills in JS and debugging it are still developing. When I find time to develop and I get stuck I need a way to send a help and get a quick response withing minutes. A bad to follow TG-chat that many follow helps me much more than a Reddit chat with 4 members that get read hours if not days later.

my2ct
Matthias

**Comment by johnd0e** [comment]
> but we are not developing for us, but for the users.

Ok. And what is difference?


> And If IITC-CE keeps to be a niche solution, not broadly accepted as the future IITC successor [...]

Does IITC has any other successor?


> When I find time to develop and I get stuck I need a way to send a help and get a quick response withing minutes.

I understand. But currently there is no any 'bad to follow TG-chat **that many follow**'.
And there never been any other chat or whatever.

Do you remember good old broadly accepted G+ channel?
There was no activity in it. Nobody posted there: neither users, nor developers.
IITC itself is niche solution, so our community never be large enough to provide immediate support like you wish.

Anyway, as we have TG channel - why not attach related TG chat, I agree.
And if it will appear more convenient than Github/Reddit, then...

**Comment by MysticJay** [comment]
1) The difference is the number of agents and operators using it.
2) IITC successors: iitc-mobile, iitczs.
![grafik](https://user-images.githubusercontent.com/25123137/61856299-25a57680-aec2-11e9-82c6-a50a858be1e5.png)
3) Many agents still fear to get a ban when using IITC. As G+ was directly linked to their Ingress account it would have been easy to ban just from being a member in that group. I guess there were many more silent readers.

Yes, please provide a name to that Chat, a public link might be a bad idea due to spamers using it.

**Comment by johnd0e** [comment]
> 1\. The difference is the number of agents and operators using it.

Perhaps there are a lot. But I still not get what are you trying to say.


> 2\. IITC successors: the italians are working on something that I know of.

It's interesting. Is that work public? Could you provide some links?

**Comment by johnd0e** [comment]
> Yes, please provide a name to that Chat

`https://t.me/iitc_group`
